### PR TITLE
Move 'Migrating from Travis' to new doc

### LINKED ISSUE #104: Consider moving "Migrate from Travis" to it's own doc
Consider moving "Migrate from Travis" to it's own doc.

Rationale:
1. It's irrelevant for 99% of people wanting to learn about our CI, but makes the doc feel longer/scarier by being there.
2. At the top we link to it straight away for people who want to learn about it - having a specific doc for it feels like a nice optimisation to send them to.

### PR DESCRIPTION
**Pull request by SaschaMann**
fixes #104

### CODE REVIEW

File: building/tracks/ci/workflow-templates.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ It will:
 
 Example implementation of these workflow files [can be found in `exercism/javascript`][git-javascript].
 
-## **HELP**: this looks like _a lot_ of work :sweat:
+## **HELP**: this looks like _a lot_ of work 😓
 
-The rest of the document is designed to explain how the workflows work. If you're in a hurry, and you just want to switch from Travis or Circle to GHA without optimizing the PR scripts, scroll down to the **~10 minute guide** on [**Migrating from Travis**](#migrating-from-travis).
+The rest of the document is designed to explain how the workflows work. If you're in a hurry, and you just want to switch from Travis or Circle to GHA without optimizing the PR scripts, go to the **~10 minute guide** on [**Migrating from Travis**](migrating-from-travis.md).
```
Comment (iHiD) [inline building/tracks/ci/workflow-templates.md]:
```suggestion
The rest of the document is designed to explain how the workflows work. If you're in a hurry, and you just want to switch from Travis or Circle to GHA without optimizing the PR scripts, check out our **~10 minute guide** on [**Migrating from Travis**](migrating-from-travis).
```

(It's not an md doc, it's html)

**Comment by iHiD** [comment]
Worked. Thanks :)

**Comment by iHiD** [comment]
(FYI: https://github.com/exercism/website/issues/686)

**Comment by SaschaMann** [comment]
Can the order in the sidebar be set manually?

I think it would make more sense to have the general Workflow Templates guide above the Travis migration guide:

grafik

**Comment by iHiD** [comment]
Yeah, at some point it will respect the order of the configs. (@ErikSchierboom You can reuse the code for the `exercises` in the main `config.json` here too)

**Pull request merged**